pkg/controllers/state/cluster.go

```diff
@@ -505,6 +505,11 @@ func (c *Cluster) populateInflight(ctx context.Context, n *StateNode) error {
 	if n.inflightInitialized {
 		return nil
 	}
+	// If the node ies already initialized, we don't need to populate its inflight capacity
```

Comment:
nit on commit but you don't need to fix it. 
```suggestion
	// If the node is already initialized, we don't need to populate its inflight capacity
```